test/Microsoft.Framework.Runtime.Roslyn.Tests/project.json

```diff
@@ -1,14 +1,17 @@
 {
-    "dependencies": {
-        "Microsoft.Framework.Runtime.Interfaces": "1.0.0-*",
-        "Microsoft.Framework.Runtime.Roslyn": "1.0.0-*",
-        "xunit.runner.aspnet": "2.0.0-aspnet-*"
-    },
-    "frameworks": {
-        "dnx451": { },
-        "dnxcore50": { }
```

Comment:
Remove `dnxcore50` since `Moq` is not supported there. For unit test, the _framework_ doesn't matter.